diagnostic.rb

```diff
@@ -10,7 +12,12 @@
 end
 
 # question 4:
+Licensee.where hair_color: %w(Hazel), eye_color: %(Black Brown)
 
 # question 5:
+Licensee.where surname: %w(Escobar Whaley) given_name: %w(Marilynn Chris)
+rails g migration ChangeHeightToOneInchShorter height:height-1
 
 # question 6:
+RichAndTeresita = Licensee.where surname: %w(Rich Myers) given_name: %w(Dylan Teresita)
+RichAndTeresita.destroy
```

Comment:
This, however, is the right idea. Once you've got the collection of matching results, you can just call `destroy` on them.